### DISCUSSION TITLE
YML ERROR / install freezes

### DISCUSSION THREAD

**Discussion by punkyard** [nextcloud/all-in-one#2022] [Questions] [answered]
### Steps to reproduce on debian 11 VPS + docker 20,10,5 compose 1,25,3 
1. create yml file in /home/docker
2. copy/paste content from github nextcloud/allinone docker-compose.yml file
3. launch docker-compose up

ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/

ERROR: The Compose file './docker-compose.yml' is invalid because:
volumes.nextcloud_aio_mastercontainer value Additional properties are not allowed ('name' was unexpected)

### Expected behavior 
install nc aio

### Actual behavior 
ERROR

#### Nextcloud AIO version 
latest v4,3,4

#### Current channel 
?

#### Other valuable info 
changing version to "3"
commenting mastercontainer name 
started the install
and freezed after:
```
nextcloud-aio-mastercontainer | {"level":"info","ts":1676745105.0171518,"msg":"serving initial configuration"}
nextcloud-aio-mastercontainer | {"level":"info","ts":1676745105.0173395,"logger":"tls.cache.maintenance","msg":"started background certificate maintenance","cache":"0xc00040fdc0"}
nextcloud-aio-mastercontainer | {"level":"info","ts":1676745105.0174718,"logger":"tls","msg":"cleaning storage unit","description":"FileStorage:/mnt/docker-aio-config/caddy/"}
nextcloud-aio-mastercontainer | {"level":"info","ts":1676745105.0175056,"logger":"tls","msg":"finished cleaning storage units"}

```
Could one telle if i'm doing wrong, or if something goes wrong?
Thanks

**Comment by Zoey2936** [comment, marked as answer]
please update to docker compose v2, docker compose v1 is more than outdated

**Comment by punkyard** [comment]
Gee! how did that happen?
thanks @Zoey2936
